Dockerfile

```diff
@@ -14,18 +14,17 @@
 
 # syntax = docker/dockerfile:1-experimental
 
-FROM --platform=${BUILDPLATFORM} golang:1.15 as base
+FROM  golang:1.15 as base
 WORKDIR /src
 ENV CGO_ENABLED=0
-COPY go.* .
+COPY go.* ./
 RUN go mod download
-COPY . .
+COPY . ./
 
 FROM base AS build
 ARG TARGETOS
 ARG TARGETARCH
-RUN --mount=type=cache,target=/root/.cache/go-build \
-  GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o /out/scorecard .
+RUN --mount=type=cache,target=/root/.cache/go-build  go build -o /out/scorecard .
```

Comment:
I don't know the impact of these changes. @dlorenc can you review this PR.

Reply:
Just to make sure I understand, the build args aren't needed here because they're auto set as env vars? Or because we're just only building one platform?

Reply:
Because we are building for only one platform. That is the reason.